frontend/dockerfile/docs/syntax.md

```diff
@@ -372,5 +372,6 @@ RUN --security=insecure cat /proc/self/status | grep CapEff
 * `BUILDKIT_MULTI_PLATFORM=<bool>` opt into determnistic output regardless of multi-platform output or not
 * `BUILDKIT_SANDBOX_HOSTNAME=<string>` set the hostname (default `buildkitsandbox`)
 * `BUILDKIT_SYNTAX=<image>` set frontend image
+* `SOURCE_DATE_EPOCH` set the UNIX timestamp for created image and layers. More info from [reproducible builds](https://reproducible-builds.org/docs/source-date-epoch/).
```

Comment:
Probably better to clarify the syntax version (1.5, unreleased)

Reply:
Updated